.github/actions/junit2jira/action.yaml

```diff
@@ -32,14 +39,29 @@ runs:
   - name: Check files
     shell: bash
     env:
+      GCP_SERVICE_ACCOUNT_STACKROX_CI: ${{ inputs.gcp-account }}
       JIRA_TOKEN: ${{ inputs.jira-token }}
     run: |
+      extra_args=()
+      if [[ "${{ inputs.create-jiras }}" == "false" ]]; then
+        extra_args=(--dry-run)
+        echo "Will use junit2jira to create CSV for BigQuery input"
```

Comment:
This makes it look like in the other branch the CSV will not be created, but the save_test_metrics call makes it seem like it will.
So which is it? Should this message be unconditional?

Reply:
I removed that line
should I remove below line too
https://github.com/stackrox/stackrox/blob/1a6d70eb0feee21a6f8638818fd95bfec718d42f/scripts/ci/lib.sh#L1195